src/DynatraceMetricExporter.ts

```diff
@@ -329,5 +341,11 @@ export class DynatraceMetricExporter implements PushMetricExporter {
 }
 
 function dimensionsFromPoint(point: DataPoint<unknown>): Dimension[] {
-	return Object.entries(point.attributes).map(([key, value]) => ({ key, value }));
+	return Object.entries(point.attributes).map(([key, value]) => {
+		if (typeof value === "string") {
+			return { key, value: value };
+		}
+
+		return { key, value: JSON.stringify(value) };
```

Comment:
Hum not sure if we want to do this? This would allow sending dimension values other than string, which we don't support _now._ For ex in the .NET exporter we don't allow it https://github.com/dynatrace-oss/opentelemetry-metric-dotnet/blob/1593bfc691be22b1623462636b0c3d934d5df69e/src/Dynatrace.OpenTelemetry.Exporter.Metrics/DynatraceMetricsExtensions.cs#L169

Reply:
Good catch! For some reason I thought that serializing it is what we do, but we should actually drop. I added the code to drop these dimensions in [a6b8d1c](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/33/commits/a6b8d1cb63d26953179a8934718a335a8bbc90b5). Thanks! :slightly_smiling_face:

Reply:
Changed it again to only allow `string` in [91e533c](https://github.com/dynatrace-oss/opentelemetry-metric-js/pull/33/commits/91e533caaf3452b2682e34be7ae757789447012a) :slightly_smiling_face: